### ISSUE TITLE
Improve visually impaired functionality

### ISSUE THREAD

**Issue by stechkev** [willowtreeapps/eyespeak-ios#281]
Here's a review we got from a german user that inspired us to think about this issue:
"the app is not barrier-free for blind and visually impaired people. Blind and visually impaired people usually work with VoiceOver. Many fields and buttons in the app as well as some of the text cannot be read under VoiceOver. Apple gives blind people and disabled people the opportunity to use many apps via VoiceOver. Unfortunately, this app is not or only partially possible with this app. Hence only one star! As soon as the app is completely accessible, this will be checked again."

Brainstorming ideas to help:
- Buttons missing VoiceOver 
- Infinite scrolling disable VoiceOver
- Navigate into and out of pages for categories. Need to let user know location
- VoiceOver voice needs to speak differently from vocable voice

**Comment by stechkev** [comment]
Lauren took a preliminary look at the app:
The most important thing I got from looking into this is that there should be some easy-to-access mechanism for the user to pause/stop the audio within the app itself. VoiceOver can be quickly disabled on iPhones. Having a quick and easy way to disable the Vocable voice gives the user control over what voice they want for the situation. If they need the Vocable app the read out texxt, they can turn off VO. If they need VO to read out the text, they can tell Vocable to not read out anything.
Here's some info. It's web focused, but the same principle applies! https://www.w3.org/WAI/WCAG21/Understanding/audio-control.html
